File: app/src/main/java/phone/air/ten/com/air_phone/bean/HttpResponse.java
package phone.air.ten.com.air_phone.bean;

import android.support.annotation.NonNull;

/**
 * Http调用响应
 */
public class HttpResponse {

    private Integer code;
    private String msg;
    private String data;

    @NonNull
    @Override
    public String toString() {
        return "HttpResponse{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                ", data='" + data + '\'' +
                '}';
    }

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }
}

File: app/src/main/java/com/ten/air/phone/model/HttpResponse.kt
package com.ten.air.phone.model

/**
 * Http调用响应
 */
class HttpResponse {

    var code: Int? = null
    var msg: String? = null
    var data: String? = null

    override fun toString(): String {
        return "HttpResponse(code=$code, msg=$msg, data=$data)"
    }

}
